mkdocs.yml

```diff
@@ -14,7 +14,7 @@ nav:
       - Getting started:
           - Installation: setup.md
           - Usage: usage.md
-      - Example workflow: notebooks/pk_wishful_workflows.ipynb
+      - Recipes: experiments.md
```

Comment:
The title and file name are different, can you make them the same?
Either rename file to recipes.md or use Experiments as title.